### PR TITLE
Convert kubernetes-e2e-upgrades-image-gke 1.4-->1.5 jobs

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230 #1866 #1871 #1881 

This PR should be merged after #1881.

/cc @krzyzacy

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-1.5-upgrade-cluster-new.env [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest-1.5 --upgrade-image=container_vm
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-1.5
+JENKINS_PUBLISHED_VERSION=ci/latest-1.4
+JENKINS_USE_SKEW_TESTS=true
+KUBE_GKE_IMAGE_TYPE=container_vm
+PROJECT=gke-up-c1-4-c1-5-up-clu-n
+ZONE=us-central1-c
+
+### version-env
+ZONE=us-central1-a
```
Comment (xiangpengzhao) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-1.5-upgrade-cluster-new.env]: Keep this one and delete `ZONE=us-central1-c`? @krzyzacy

Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-1.5-upgrade-cluster-new.env]: yeah kill the previous line

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

File: jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-1.5-upgrade-cluster-new.env (hunk 1 above)
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-1.5-upgrade-cluster-new.env]: I'll let you fix #1881 and rebase this one

**Comment by xiangpengzhao** [comment]
Addressed. PTAL @krzyzacy . But the Bazel test failed, not sure why yet.

**Comment by xiangpengzhao** [comment]
Oh, I think it would be successful after #1881 merged.

**Comment by krzyzacy** [comment]
yeah, merged #1881, please rebase this and the other ones :-)

**Comment by xiangpengzhao** [comment]
Rebased. PTAL @krzyzacy

**Pull request merged**